CONTRIBUTING.md

```diff
@@ -17,19 +84,219 @@ augroup gno_autocmd
 augroup END
 ```
 
-TODO: other vim tweaks to make work with vim-go etc.
-
-### Getting started with Emacs
+#### Emacs Support
 
-Install [go-mode.el](https://github.com/dominikh/go-mode.el).
-
-Add to your emacs configuration file:
+1. Install [go-mode.el](https://github.com/dominikh/go-mode.el).
+2. Add to your emacs configuration file:
 
 ```lisp
 (add-to-list 'auto-mode-alist '("\\.gno\\'" . go-mode))
 ```
 
-## Notable Contributions
+### Local Setup
+
+To get started with Gno development, the process is relatively straightforward.
+
+Clone the repo:
+`git clone https://github.com/gnolang/gno.git`
+
+Build / install base commands:
+`make build `
+
+That’s it!
+
+### Testing
```

Comment:
Can we also mention `gnodev test` here? It's very helpful for testing gno smart contracts

Reply:
I love this idea 💯 

Added:

[93d6736](https://github.com/gnolang/gno/pull/469/commits/93d6736e126be461a3246c583b53c1a57367cd0b)